pkg/controller/storagecluster/placement.go

```diff
@@ -33,13 +33,6 @@ func getPlacement(sc *ocsv1.StorageCluster, component string) rookv1.Placement {
 		}
 	}
 
-	topologyMap := sc.Status.NodeTopologies
-	if topologyMap != nil && (component == "mon" || component == "mds") {
-		topologyKey := determineFailureDomain(sc)
-		topologyKey, _ = topologyMap.GetKeyValues(topologyKey)
```

Comment:
@jarrpa Once we determine the topologyKey for the StorageCluster, what is the reason to get the keyvalue pairs but discard the values?

Is is possible that the topologyKey obtained using determineFailureDomain function is a substring of the "real" key?